src/cascadia/TerminalApp/LeafPane.cpp

```diff
@@ -0,0 +1,717 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "LeafPane.h"
+#include "AppLogic.h"
+
+#include <Mmsystem.h>
+
+#include "LeafPane.g.cpp"
+
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Graphics::Display;
+using namespace winrt::Windows::UI;
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Windows::UI::Xaml::Media;
+using namespace winrt::Microsoft::Terminal::Settings::Model;
+using namespace winrt::Microsoft::Terminal::TerminalControl;
+using namespace winrt::Microsoft::Terminal::TerminalConnection;
+using namespace winrt::TerminalApp;
```

Comment:
```suggestion
```
nit: I think you can just remove this one since we're entirely in the `winrt::TerminalApp::implementation` namespace